src/main/java/com/sulphate/chatcolor2/main/MainClass.java

```diff
@@ -158,6 +157,35 @@ private void setupDefaultConfig() {
         defaultConfig.put("messages.failed-to-enable", "Failed to enable the plugin. Please try again, making sure all folders are not locked.");
         defaultConfig.put("messages.successfully-enabled", "The plugin has been successfully enabled.");
         defaultConfig.put("messages.already-enabled", "The plugin is already enabled.");
+        defaultConfig.put("messages.black", "Black");
```

Comment:
Love the new configurable messages, thanks :)

Reply:
I'm glad you like them :)